samples/test.py

```diff
@@ -1,72 +1,82 @@
-	and calculate their energies and forces.
-	"""
-	nsamp = 10000
-	crds = np.random.uniform(4.0,size = (nsamp,3,3))
-	st = MSet()
-	MDL = None
-	natom = 4
-	ANS = np.array([3,1,1])
-	if (model_=="Morse"):
-		MDL = MorseModel()
-	else:
-		MDL = QuantumElectrostatic()
-	for s in range(nsamp):
-		if (model_=="Morse"):
-			st.mols.append(Mol(np.array([1.,1.,1.]),crds[s]))
-			en,f = MDL(crds[s])
-			st.mols[-1].properties["dipole"] = np.array([0.,0.,0.])
-		else:
-			st.mols.append(Mol(np.array([3.,1.,1.]),crds[s]))
-			en, f, d, q = MDL(crds[s])
-			st.mols[-1].properties["dipole"] = d
-			st.mols[-1].properties["charges"] = q
-		st.mols[-1].properties["energy"] = en
-		st.mols[-1].properties["force"] = f
-		st.mols[-1].properties["gradients"] = -1.0*f
-		st.mols[-1].CalculateAtomization()
-	return st
+def GenerateData(model_=None):
+    """
+    Generate random configurations in a reasonable range.
+    and calculate their energies and forces.
 
-def TestTraining_John():
-	PARAMS["train_dipole"] = True
-	tset = GenerateData()
-	net = BehlerParinelloDirectGauSH(tset)
-	net.train()
-	return
+    Args:
+        model (str): Type of model to be used. If "Morse is supplied, a morse potential will be
+                     used to create the dataset (no dipole moments or charge). If anything else
+                     is supplied, a Quantum Electrostatic potential will be used to create the
+                     dataset, which will include dipole moments and charge. In either case,
+                     energy, forces, and gradients are all calculated.
+    """
+    # Configuration
+    nsamp = 10000 # Number of configurations to sample
+    coords = np.random.uniform(low=1.0, high=4.0, size = (nsamp,3,3)) # Generate random 3D coordinates for nsamp atoms
 
-def TestTraining():
-	a = GenerateData()
-	TreatedAtoms = a.AtomTypes()
-	PARAMS["NetNameSuffix"] = "training_sample"
-	PARAMS["learning_rate"] = 0.00001
-	PARAMS["momentum"] = 0.95
-	PARAMS["max_steps"] = 15 # Train for 5 epochs in total
-	PARAMS["batch_size"] =  100
-	PARAMS["test_freq"] = 5 # Test for every epoch
-	PARAMS["tf_prec"] = "tf.float64" # double precsion
-	PARAMS["EnergyScalar"] = 1.0
-	PARAMS["GradScalar"] = 1.0/20.0
-	PARAMS["NeuronType"] = "sigmoid_with_param" # choose activation function
-	PARAMS["sigmoid_alpha"] = 100.0  # activation params
-	PARAMS["KeepProb"] = [1.0, 1.0, 1.0, 1.0] # each layer's keep probability for dropout
-	d = MolDigester(TreatedAtoms, name_="ANI1_Sym_Direct", OType_="AtomizationEnergy")
-	tset = TensorMolData_BP_Direct_EandG_Release(a, d, order_=1, num_indis_=1, type_="mol",  WithGrad_ = True)
-	manager=TFMolManage("",tset,False,"fc_sqdiff_BP_Direct_EandG_SymFunction")
-	PARAMS['Profiling']=0
-	manager.Train(1)
+    # Initialize our dataset
+    dataset = tm.MSet()
+    # Morse -> Energy / Forces
+    if model_ == "Morse":
+        MDL = tm.MorseModel()
+        atomic_numbers = np.array([1,1,1], dtype=np.uint8) # Hydrogen / Hydrogen / Hydrogen
```

Comment:
The file is not PEP8 formatted ( there should be spaces after array members) - use re-indent in PyCharm pls.

Reply:
Autoformatted in PyCharm